sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -230,6 +261,9 @@ Le service %1 n'a pas réussi à installer une source d'évènement.
 Language=Polish
 Usługa %1 nie mogła skonfigurować źródła zdarzenia.
 .
+Language=Romanian
+Serviciul %1 nu a putut configura o sursă a evenimentului.
```

Comment:
English test says "an event", whereas the Romanian translates it as "the event".
...but maybe this is not that important.